src/functions/TestDrive.ps1

```diff
@@ -1,10 +1,29 @@
 ﻿function Get-TestDrivePlugin {
-    New-PluginObject -Name "TestDrive" -Start {
+    $p = @{
+        Name = 'TestDrive'
+    }
+
+    $p.Start = {
+        param($Context)
+
         if (& $script:SafeCommands['Test-Path'] TestDrive:\) {
-            & $SafeCommands['Remove-Item'] (& $SafeCommands['Get-PSDrive'] TestDrive -ErrorAction Stop).Root -Force -Recurse -Confirm:$false
```

Comment:
Root does not include provider, so this used `HKEY_CURRENT_USER\Software\Pester` as a relative filepath. Likely never worked.